README.md

```diff
@@ -4,95 +4,53 @@
 
 # IPLD
 
-[![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://ipn.io)
-[![](https://img.shields.io/badge/project-ipld-blue.svg?style=flat-square)](http://github.com/ipld/ipld)
-[![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
+[![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](https://protocol.ai)
+[![](https://img.shields.io/badge/project-ipld-blue.svg?style=flat-square)](https://github.com/ipld/ipld)
+[![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](https://webchat.freenode.net/?channels=%23ipfs)
 
 > InterPlanetary Linked Data
 
-Read more about IPLD at https://ipld.io
+Captain: [@nicola](https://github.com/nicola)
 
-_Note that, as in many IPFS repositories, most of the work is happening in [the issues](https://github.com/ipld/ipld/issues/) or in [active pull requests](https://github.com/ipld/ipld/pulls/). Go take a look!_
 
-### IPLD Definitions
-
-There are a variety of systems that use merkle-tree and hash-chain inspired datastructures (e.g. git, bittorrent, IPFS, tahoe-lafs, sfsro). IPLD defines:
-
-- merkle-links: the core unit of a merkle-graph
-- merkle-dag: any graphs whose edges are merkle-links.
-- merkle-paths: unix-style paths for traversing merkle-dags with named merkle-links
-- IPLD Data Model: a flexible, JSON-based data model for representing merkle-dags.
-- IPLD Serialized Formats: a set of formats in which IPLD objects can be represented, for example JSON, CBOR, CSON, YAML, Protobuf, XML, RDF, etc.
-- IPLD Canonical Format: a deterministic description on a serialized format that ensures the same logical object is always serialized to the exact same sequence of bits. This is critical for merkle-linking, and all cryptographic applications.
```

Comment:
So, this information can actually be quite useful. Personally, I'd get rid of the last point (the concept "canonical format" is a bit in flux) and change "a flexible, JSON-based data model for representing merkle-dags" to "a flexible, JSON-inspired, self-describing structured data model for representing merkle-dags." (at the risk of being a bit wordy).

Probably in a section at the bottom. I believe this list grew out of a lot of repeated confusion/questions so getting rid of it wholesale may not be the best idea.

Reply:
I opted to remove it while the spec work is still ongoing, to have this place be mostly centered around pointing directions. Are all these terms accurate?

(Agreed about all other comments.)

Reply:
Except for the pieces I pointed out, yes. We still talk about merkle-links, merkle-dags, merkle-paths, and serialization formats. Personally, I'd keep this and frame it as an FYI. That is, "As you read through documents related to IPLD, you may come across the following terms:".